src/main/java/org/terasology/logic/health/event/RegenerateEvent.java

```diff
@@ -0,0 +1,63 @@
+/*
```

Comment:
Currently, this event is not used. After the discussion on how to do regen is concluded, will update/use this in the authority system.